Feature/border width

YinDongFang opened the pull request:
1. Support borderWidth
2. Only show grid line in rounded area

![image](https://github.com/user-attachments/assets/d9b56344-9b7e-442e-89b9-8c5fdbe435c3)

YinDongFang commented:
@mosch Please take a moment to review. Thanks.

mosch commented:
I don't understand why that is useful? What benefit does this have over wrapping the Editor into a div with a border?

YinDongFang commented:
Thanks for your review!
I'm using this great library in my project. Border and grid style changes were requested by our UI as new requirements. I found the borderColor property was already support by previous PR, but can't set width. So I implemented borderWidth as optional properties. I also adjusted grid line style, I think it's looks better in circle shape mask. If used as rect shape, it also can display use borderColor and borderWidth.
